src/coda/apps/invoices/views/inspect.py

```diff
@@ -214,6 +233,22 @@ def position_viewmodel(position: Position[ItemType], number: int) -> "PositionVi
     )
 
 
+@login_required
+@require_POST
+def pay_invoice(request: HttpRequest, pk: int) -> HttpResponse:
```

Comment:
we should consider just reloading the entire page here. we can use `redirect` for that